candidate_bestPractice_OrangeHRM/cypress/e2e/OrangeHRM/candidate.spec.cy.ts

```diff
@@ -4,17 +4,18 @@ import candidate from "../../support/projectObject/candidate";
 const candidateObj: candidate = new candidate();
 
 describe("OrangeHRM-create and edit vacancy", () => {
-  
   beforeEach(function () {
     cy.visit("/");
     candidateObj.loginFUNC("Admin", "admin123");
-    apiHelpers.addCandidate()
+    apiHelpers.addCandidate();
+  });
 
-      
+  afterEach(function () {
+    apiHelpers.deleteCandidate();
   });
 
   it("TCs2: OrangeHRM-edit vacancy", () => {
-    apiHelpers.stateShortlistStatus()
-    candidateObj.scheduleInterview()
+    apiHelpers.stateShortlistStatus();
+    candidateObj.scheduleInterview();
   });
 });
```

Comment:
Where is the assertion for this scenario?

Reply:
you mean that I have to add the following code to assert the shortlist update:
```
.then((response)=>{
      expect(response.status).eq(200)

    })
```

And the following code to assert scheduling an interview:
```
cy.get('.oxd-text.oxd-text--p.oxd-text--subtitle-2').should('contain','Status: Interview Scheduled')
```

Reply:
Yes, I mean the second code to assert the status that should be Interview Scheduled, so you should make a function for this assertion and call it in the test